astroid/node_classes.py

```diff
@@ -2567,11 +2567,14 @@ class Const(mixins.NoChildrenMixin, NodeNG, bases.Instance):
 
     _other_fields = ("value",)
 
-    def __init__(self, value, lineno=None, col_offset=None, parent=None):
+    def __init__(self, value, kind=None, lineno=None, col_offset=None, parent=None):
```

Comment:
Technically this is the right position (after `value`). I was just thinking that this could break calls which don't use keyword arguments. Would it be better to add `kind` as last argument? @hippo91

In the long term it's probably a good idea to add `*` and change to keyword only arguments.

Reply:
I agree with you @cdce8p . Putting the new parameter at the end is safer.

Reply:
@federicobond Would you mind changing it then? Since `kind` is an optional argument, you should be able to get rid of some `None` arguments with that.

Reply:
Should be fixed now.

Reply:
Thanks @federicobond!